exec/csnex/Exec.cs

```diff
@@ -11,27 +11,32 @@ public Executor(Bytecode bc)
         {
             exit_code = 0;
             stack = new Stack<Cell>();
+            Callstack = new Stack<int>();
             global = new Global(this);
             bytecode = bc;
+            ParamRecursionLimit = 2000; // ToDo: Add runtime$setRecursionLimit()
         }
 
         private int exit_code;
         private readonly Bytecode bytecode;
         public Stack<Cell> stack;
+        private Stack<int> Callstack;
```

Comment:
The capitalization style here seems inconsistent. Should this be `callstack`?

Reply:
Fixed.